shared/templates/package_removed/bash.template

```diff
@@ -4,10 +4,12 @@
 # complexity = low
 # disruption = low
 
-# CAUTION: This remediation script will remove {{{ PKGNAME }}}
-#	   from the system, and may remove any packages
-#	   that depend on {{{ PKGNAME }}}. Execute this
-#	   remediation AFTER testing on a non-production
-#	   system!
+# CAUTION: This remediation script will remove {{{ PACKAGES | join(" and ") }}}
```

Comment:
This might not look the best if we every have 3 packages to remove, but I think we can live with it for now.